lib/smart_proxy_openscap/openscap_plugin.rb

```diff
@@ -18,6 +18,7 @@ class Plugin < ::Proxy::Plugin
     https_rackup_path File.expand_path("http_config.ru", File.expand_path("../", __FILE__))
 
     default_settings :spooldir => '/var/spool/foreman-proxy/openscap',
-                     :openscap_send_log_file => '/var/log/foreman-proxy/openscap-send.log'
+                     :openscap_send_log_file => '/var/log/foreman-proxy/openscap-send.log',
+                     :contentdir => '/var/lib/openscap/content'
```

Comment:
hm, I didn't see the previous discussion, if we create symlink that leads to `/var/lib` during packaging then it's ok and you can dismiss my comment, just note that without the symlink the relative path would be /usr/share/foreman-proxy on rpm based distros, which is owned by root and storing dynamic content there is not a good idea